src/auro-menu.js

```diff
@@ -210,7 +205,7 @@ class AuroMenu extends LitElement {
    * Process actions for making making a menuoption selection.
    */
   makeSelection() {
-    if (!this.items) {
+    if (this.items.length === 0) {
```

Comment:
The condition is `(!this.items)` before it was changed by me, which means it can have undefined, okay.
After going through the code several times, I couldn't find a case or a timing, where `this.items` value has undefined value instead of empty array, when `makeSelection` function is invoked.

Also, isn't this 'preset value' case that you mentioned is going to trigger 'selectByValue' handler instead? I did nothing to that part.

Please advise.

Reply:
I found a way to recreate the case in the test file and managed to cover the code for this.